st2actions/st2actions/notifier/notifier.py

```diff
@@ -50,8 +51,8 @@
 
 LOG = logging.getLogger(__name__)
 
-ACTIONUPDATE_WORK_Q = liveaction.get_queue('st2.notifiers.work',
-                                           routing_key=publishers.UPDATE_RK)
+ACTIONUPDATE_WORK_Q = execution.get_queue('st2.notifiers.work',
```

Comment:
Hm, I think this should be backward compatible (no messages will be lost and there should be no duplication) because of the way how live actions and executions are updated, right?